src/cpp/src/visual_language/inputs_embedder.cpp

```diff
@@ -194,8 +199,8 @@ class InputsEmbedder::IInputsEmbedder {
             // so let's check it out, find the trusted part and use it in on the next step
             size_t trusted_history_length = 0;
             if (!m_tokenized_history.empty()) {
-                std::set<int64_t> stop_tokens = {m_tokenizer.get_eos_token_id()};
-                trusted_history_length = ov::genai::utils::get_first_history_difference(prev_chat_tokens, m_tokenized_history, stop_tokens);
+                OPENVINO_ASSERT(!m_stop_token_ids.empty(), "Stop tokens are not set for InputsEmbedder");
```

Comment:
Why should it be raised error without stop_token_ids? Generation could run and finished without it

Reply:
We do not have generation config in InputsEmbedder, but we do in VLM pipeline (as well as in other pipelines).
Previously we was relying on single stop token retrieved from tokenizer:
```
std::set<int64_t> stop_tokens = {m_tokenizer.get_eos_token_id()};
```
Now, when several stop tokens can present in generation config, we need to ensure that the same stop tokens are used in InputsEmbedder. For this I am using `m_stop_token_ids` field. This assert checks that we did not forget to call `set_stop_token_ids()` in VLM pipeline for InputsEmbedder.
Does it make sense or it is still not needed here? @sbalandi @ilya-lavrenov

Reply:
yes, but if I understand right it is possible to set generation config that stop_token_ids is empty , in this case it will fail incorrectly

Reply:
agree with @sbalandi 
according to our tests 
https://github.com/openvinotoolkit/openvino.genai/blob/38ab05522fa661de3df32fd95296fe54f9252f4e/tests/python_tests/test_generation_config.py#L25-L26

absence of any stop token is valid scenario.

Reply:
Removed